src/xml-properties-mixin.js

```diff
@@ -61,8 +61,9 @@ let XmlPropertiesMixin = (superclass) => class extends superclass {
       ({ name }) => !isPropertiesTag(name)
     );
     if ($Pr.length) {
-      // once our $Pr has been created, it shouldn't change, so we'll cache it
-      this._Pr = $Pr;
+      // once our $Pr has been created, it shouldn't change, so we'll replace
+      // this getter with a simple property
+      Object.defineProperty(this, `$Pr`, { value: $Pr });
```

Comment:
Can you leave some kind of comment about the expected lifetime of objects using this mixin or something? Just reading the code, it looks like if you ever call `removePr()` you perma-bork your object because the removed `$Pr` will remain cached forever.

Reply:
Hmm. Good point.